src/fastoad/gui/analysis_and_plots.py

```diff
@@ -419,50 +450,39 @@ def mass_breakdown_sun_plot(
             variables, var_names_and_new_units
         )
 
-    # pylint: disable=unbalanced-tuple-unpacking # It is balanced for the parameters provided
-
     # TODO: Deal with this in a more generic manner ?
     if round(tow, 6) == round(owe + payload + onboard_fuel_at_takeoff, 6):
         tow = owe + payload + onboard_fuel_at_takeoff
 
-    fig = make_subplots(1, 2, specs=[[{"type": "domain"}, {"type": "domain"}]])
-
-    fig.add_trace(
-        go.Sunburst(
-            labels=[
-                tow_name + "<br>" + str(int(tow)) + " [kg]",
-                "payload"
-                + "<br>"
-                + str(int(payload))
-                + " [kg] ("
-                + str(round(payload / tow * 100, 1))
-                + "%)",
-                "onboard_fuel_at_takeoff"
-                + "<br>"
-                + str(int(onboard_fuel_at_takeoff))
-                + " [kg] ("
-                + str(round(onboard_fuel_at_takeoff / tow * 100, 1))
-                + "%)",
-                "OWE" + "<br>" + str(int(owe)) + " [kg] (" + str(round(owe / tow * 100, 1)) + "%)",
-            ],
-            parents=[
-                "",
-                tow_name + "<br>" + str(int(tow)) + " [kg]",
-                tow_name + "<br>" + str(int(tow)) + " [kg]",
-                tow_name + "<br>" + str(int(tow)) + " [kg]",
-            ],
-            values=[tow, payload, onboard_fuel_at_takeoff, owe],
-            branchvalues="total",
+    labels = [
+        _get_sunburst_mass_label(tow_name, tow),
+        _get_sunburst_mass_label("payload", payload, parent_value=tow),
+        _get_sunburst_mass_label(
+            "onboard fuel at takeoff", onboard_fuel_at_takeoff, parent_value=tow
         ),
-        1,
-        1,
+        _get_sunburst_mass_label("OWE", owe, parent_value=tow),
+    ]
+
+    return go.Sunburst(
+        labels=labels,
+        parents=["", labels[0], labels[0], labels[0]],
+        values=[tow, payload, onboard_fuel_at_takeoff, owe],
+        branchvalues="total",
     )
 
+
+def _get_OWE_sunburst_plot(variables: VariableList):
+    """
+
```

Comment:
Why empty line?

Reply:
> Everything looks fine and tidy to me. I just want to clarify the small formatting inconsistency before merging, as all of the other file of the repo have been formatted differently. Thanks for the PR!

Indeed, these 2 empty lines had no justification 😄. Fixed in d023e18819ffb13522f87b4336e8dc16e401f1b7.